src/templates/static/legal/tac_sections/tac.jsx

```diff
@@ -3,14 +3,13 @@ import React from 'react';
 const Tac = () => (
     <div>
         <h2 data-anchor='terms-and-conditions'>{it.L('Terms and conditions')}</h2>
-        <p><i>{it.L('Version 52. Last modified 26-11-2021.')}</i></p>
+        <p><i>{it.L('Version 53. Last modified 10-02-2022.')}</i></p>
```

Comment:
Please consider, the last agreed date is 01-03-2022
it should be changed with PR https://github.com/binary-com/binary-static/pull/7055